src/cwe_checker_lib/src/analysis/expression_propagation/mod.rs

```diff
@@ -191,7 +191,7 @@ fn insert_expressions(
 ) {
     for sub in program.subs.values_mut() {
         for block in sub.term.blocks.iter_mut() {
-            block.propagate_input_expressions(inseratables.get(&block.tid).cloned());
+            propagate_input_expressions(block, inseratables.get(&block.tid).cloned());
```

Comment:
Could you also fix the typo `inseratables`instead of `insertables` here? I know, it has nothing to do with your PR, but we cannot just leave it there...